app/src/main/java/ar/com/wolox/android/example/ui/home/HomeFragment.kt

```diff
@@ -1,12 +1,46 @@
 package ar.com.wolox.android.example.ui.home
 
+import androidx.core.content.ContextCompat
 import ar.com.wolox.android.R
 import ar.com.wolox.android.databinding.FragmentHomeBinding
+import ar.com.wolox.android.example.ui.home.news.NewsFragment
+import ar.com.wolox.android.example.ui.home.profile.ProfileFragment
 import ar.com.wolox.android.example.ui.login.LoginActivity
+import ar.com.wolox.wolmo.core.adapter.viewpager.SimpleFragmentPagerAdapter
 import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import javax.inject.Inject
 
 class HomeFragment : WolmoFragment<FragmentHomeBinding, HomePresenter>(), HomeView {
-    override fun init() {}
+
+    @Inject
+    internal lateinit var newsFragment: NewsFragment
+
+    @Inject
+    internal lateinit var profileFragment: ProfileFragment
+
+    override fun init() {
+        with(binding) {
+            viewPager.adapter = SimpleFragmentPagerAdapter(childFragmentManager).apply {
+                addFragments(
+                        newsFragment to "NewsFragment",
+                        profileFragment to "ProfileFragment"
```

Comment:
Remember to add "NewsFragment" and "ProfileFragment" to a contant value in the companion